config/config.go

```diff
@@ -47,6 +47,17 @@ func NewDefaultConfig() *AgentConfig {
 				JwksURL:            "",
 				JwksUpdateInterval: 0,
 			},
+			CORS: CORSConfig{
+				// If AllowedOrigins is nil or empty, value is set to ["*"].
+				AllowedOrigins: nil,
+				// If AllowedMethods is nil or empty, value is set to (HEAD, GET and POST).
+				AllowedMethods: []string{"HEAD", "GET", "PUT", "POST", "DELETE"},
```

Comment:
Might as well default to `nil` or empty here rather than to _extend_ the default.

Reply:
Sure, will do.